closure/filegroup_external.bzl

```diff
@@ -28,11 +28,11 @@ def _filegroup_external(ctx):
     basename = ""
     for url in urls:
       basename = url[url.rindex("/") + 1:] or basename
-      if url in downloaded:
+      if url in downloaded.to_list():
```

Comment:
Wouldn't it be better to define these as lists instead of casting them every time?

Reply:
Indeed. Done.